Packs/VMRay/Integrations/VMRay/VMRay.yml

```diff
@@ -399,6 +399,33 @@ script:
     execution: false
     name: vmray-get-sample-by-hash
     outputs:
+    - contextPath: File.Name
+      description: The full file name (including file extension).
+      type: String
+    - contextPath: File.MD5
+      description: The MD5 hash of the file.
+      type: String
+    - contextPath: File.SHA1
+      description: The SHA1 hash of the file.
+      type: String
+    - contextPath: File.SHA256
+      description: The SHA256 hash of the file.
+      type: String
+    - contextPath: File.SSDeep
+      description: The SSDeep hash of the file.
+      type: String
+    - contextPath: DBotScore.Indicator
+      description: The indicator that was tested.
+      type: String
+    - contextPath: DBotScore.Type
+      description: The indicator type.
+      type: String
+    - contextPath: DBotScore.Vendor
+      description: The vendor used to calculate the score.
+      type: String
+    - contextPath: DBotScore.Score
+      description: The actual score.
+      type: Number
```

Comment:
Please add the `Reliability` output.
You can take a look here - https://xsoar.pan.dev/docs/integrations/dbot#reliability-level, https://xsoar.pan.dev/docs/integrations/context-standards-mandatory